### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by Stitch10925** [bitmagnet-io/bitmagnet#328]
- [v] I have checked the existing issues to avoid duplicates
- [v] I have redacted any info hashes and content metadata from any logs or screenshots attached to this issue

### Is your feature request related to a problem? Please describe

Add the ability to pause indexing and tracking, but keep Bitmagnet available for querying (by Prowlarr for example). This will allow users to pause indexing and prevent the database of growing further in case their hard disks are starting to become full.

### Describe the solution you'd like

A toggle inside of a settings page (maybe under the hamburger menu) which allows you to turn indexing on or off.

### Describe alternatives you've considered

Stopping the Bitmagnet Docker container will prevent the database from growing and filling up disk, but it does not allow Bitmagnet to be query-able anymore.

**Comment by mgdigital** [comment]
> Stopping the Bitmagnet Docker container will prevent the database from growing and filling up disk, but it does not allow Bitmagnet to be query-able anymore.

Yes it does - you can already control which workers start and run bitmagnet without the DHT crawler. Just start bitmagnet with `bitmagnet worker run --keys=http_server`.
